### ISSUE TITLE
Dont pollute _ namespace

### ISSUE THREAD

**Issue by declann** [calculang/calculang#60]
The reason this is a Requirement is because a model that does

Import {price_}
Price = ...price_...

Breaks the possibility of other models overriding price again.

All_cul issue should NOT import any _s (so no problem about not knowing how to as them), they should always be explicit which is logical

**Comment by declann** [comment]
Approaches combo of:

Eslint rule that must run on .cul.js files, pros highlights error in editor, prob the way to go in medium term.

Eslint rule on esm: no _ export outside of a fn declaration.

Build an error into calculang code.

Eslint rule on esm fits current linting approach best, but may be a temporary one.

**Comment by declann** [comment]
below handy config for .eslintrc.js that also closes #49 whenever an eslint check on esm output reflecting this config is integrated with calculang compilation

````
  rules: {
    'no-undef': ['error', {}],
    'no-restricted-syntax': [
      'error',
      {
        message: "calculang: don't pollute the _ namespace",
        selector:
          'ImportDeclaration[source.value=/cul_scope_/] > ImportSpecifier[local.name=/_$/]',
        // converted to esm => match cul_scope_x rather than .cul
      },
    ],
  },
```

**Comment by declann** [comment]
closing because meta issue #71 delivers this for all
